docs/ref/checks.txt

```diff
@@ -317,6 +317,9 @@ Models
   or a number.
 * **models.E034**: The index name ``<index>`` cannot be longer than
   ``<max_length>`` characters.
+* **models.W035**: ``db_table`` ``<db_table>`` is used by multiple models:
+  ``<model list>``. This may not be an issue because database routers are
+  installed.
```

Comment:
I'm not 100% happy with the wording here. 

Can't think of better at this exact moment. I'll come back to it tomorrow. In the meantime, in anyone has a nice phrasing, that'd be good.